test/libsolidity/util/TestFileParser.cpp

```diff
@@ -107,6 +108,13 @@ vector<solidity::frontend::test::FunctionCall> TestFileParser::parseFunctionCall
 						if (lowLevelCall)
 							call.kind = FunctionCall::Kind::LowLevel;
 
+						// Treat all calls to functions containing '.' as calls to builtin functions.
+						if (call.signature.find('.') != std::string::npos)
```

Comment:
Maybe it is better to try finding signature in `builtins` instead of assuming that it contains `.`?

Reply:
Ah yes, thats much better :)